examples/meta/src/classifier/multiclass_ecoc_random.sg

```diff
@@ -0,0 +1,39 @@
+CSVFile f_feats_train("../../data/classifier_4class_2d_linear_features_train.dat")
+CSVFile f_feats_test("../../data/classifier_4class_2d_linear_features_test.dat")
+CSVFile f_labels_train("../../data/classifier_4class_2d_linear_labels_train.dat")
+CSVFile f_labels_test("../../data/classifier_4class_2d_linear_labels_test.dat")
+Math:init_random(1)
```

Comment:
EDIT:
the 0 seed in Shogun causes a random seed to be generated. I didnt know that
So you have to use 1.

Also, can you send a mini PR that adds a comment in the CMath interface documentaion?